README.adoc

```diff
@@ -89,117 +61,37 @@ So for the time being, you need to:
-
-Run `nix build -f default.nix plutus.haskell.packages.plutus-core.components.library` from the root to build the Plutus Core library.
-
-See <<nix-build-attributes>> to find out what other attributes you can build.
-
-==== How to build the Haskell packages with `cabal`
-
-The Haskell packages can be built directly with `cabal`.
-We do this during development (see <<how-to-develop>>).
-The best way is to do this is inside a `nix-shell`.
-
-[NOTE]
-====
-For fresh development setups, you also need to run `cabal update`.
-====
-
-Run `cabal build plutus-core` from the root to build the Plutus Core library.
-
-See the link:./cabal.project[cabal project file] to see the other packages that you can build with `cabal`.
-
-[[nix-advice]]
-== Nix
+== Documentation
 
-[[iohk-binary-cache]]
-=== How to set up the IOHK binary caches
+=== User documentation
 
-Adding the IOHK binary cache to your Nix configuration will speed up
-builds a lot, since many things will have been built already by our CI.
+The main documentation is located https://plutus.readthedocs.io/en/latest/[here].
 
-If you find you are building packages that are not defined in this
-repository, or if the build seems to take a very long time then you may
-not have this set up properly.
+=== Talks
 
-To set up the cache:
+- https://www.youtube.com/watch?v=MpWeg6Fg0t8[Functional Smart Contracts on Cardano (2020)]: an overview of the ideas behind the Plutus Platform.
+- https://www.youtube.com/watch?v=usMPt8KpBeI[The Plutus Platform (2020)]: an overview of the Platform as a whole (including the Application Framework) at the time.
 
-. On non-NixOS, edit `/etc/nix/nix.conf` and add the following lines:
-+
-----
-substituters        = https://hydra.iohk.io https://iohk.cachix.org https://cache.nixos.org/
-trusted-public-keys = hydra.iohk.io:f/Ea+s+dFdN+3Y/G+FDgSq+a5NEWhJGzdjvKNGv0/EQ= iohk.cachix.org-1:DpRUyj7h7V830dp/i6Nti+NEO2/nhblbov/8MW7Rqoo= cache.nixos.org-1:6NCHdD59X431o0gWypbMrAURkbJ16ZPMQFGspcDShjY=
-----
-+
-[NOTE]
-====
-If you don't have an `/etc/nix/nix.conf` or don't want to edit it, you may add the `nix.conf` lines to `~/.config/nix/nix.conf` instead.
-You must be a https://nixos.org/nix/manual/#ssec-multi-user[trusted user] to do this. If this file doesn't exist, go ahead and create it.
+=== Specifications and design
 
-Remember that nix will use `~/.config/nix/nix.conf` over `/etc/nix/nix.conf` if both files exist.
-====
-. On NixOS, set the following NixOS options:
-+
-----
-nix = {
-  binaryCaches          = [ "https://hydra.iohk.io" "https://iohk.cachix.org" ];
-  binaryCachePublicKeys = [ "hydra.iohk.io:f/Ea+s+dFdN+3Y/G+FDgSq+a5NEWhJGzdjvKNGv0/EQ=" "iohk.cachix.org-1:DpRUyj7h7V830dp/i6Nti+NEO2/nhblbov/8MW7Rqoo=" ];
-};
-----
+- https://hydra.iohk.io/job/Cardano/plutus/linux.docs.plutus-report/latest/download-by-type/doc-pdf/plutus[Plutus Technical Report (draft)]: a technical report and design document for the project.
```

Comment:
We need to work out how to get these from cicero...

Reply:
That's right, ill make a Jira story